### PR TITLE
Remove deprecated always_populate_raw_post_data error

### LINKED ISSUE #3948: Feedback on page: /guides/v2.3/install-gde/trouble/php/tshoot_php-set.html
This manual mentions to uncomment the line: 
always_populate_raw_post_data

This feature was DEPRECATED in PHP 5.6.0, and REMOVED as of PHP 7.0.0. As such it doesn't exist in up-to-date installations.

Please remove any mentioning of this property. 

Thanks.

### PR DESCRIPTION
**Pull request by ronak2ram**
## This PR is a:

- Content fix or rewrite

## Summary

When this pull request is merged, it will not show deprecated variable error.
php.ini in removed variable **always_populate_raw_post_data** for the **PHP 7.0 or greater**.
http://php.net/manual/en/ini.core.php#ini.always-populate-raw-post-data

So Magento 2.2 and 2.3 support PHP 7.0 or greater so this variable(always_populate_raw_post_data) error never come to Magento v2.2 and v2.3. So I removed the solution of the variable error always_populate_raw_post_data.

Fixed #3948
In short, 

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.2/install-gde/trouble/php/tshoot_php-set.html
- https://devdocs.magento.com/guides/v2.3/install-gde/trouble/php/tshoot_php-set.html.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by dobooth** [comment]
running tests

**Comment by shrielenee** [comment]
@dobooth Ready to roll? 🎲

**Comment by dobooth** [comment]
running tests
